.github/eks-workflow-files/mpi-nccl-test.yml

```diff
@@ -0,0 +1,75 @@
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: PLACEHOLDER
+spec:
+  # Without this then the first few attempts to run the launcher will result in errors
+  # due to failed DNS resolution of the worker names. It works eventually, given a big
+  # enough backoffLimit, but it makes it harder to handle log-streaming and identifying
+  # the "real" exit code of the job.
+  launcherCreationPolicy: WaitForWorkersReady
+  runPolicy:
+    cleanPodPolicy: Running
+    # surface errors direct to GitHub Actions without internal retries
+    backoffLimit: 0
+  # 1 MPI rank per GPU
+  slotsPerWorker: 8
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      # Without this the launcher pod will be deleted on failure, which makes it hard
+      # to provide useful diagnostics
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+            - image: PLACEHOLDER
+              imagePullPolicy: IfNotPresent
+              name: PLACEHOLDER
+              command:
+                - mpirun
```

Comment:
Does YAML permit the `[...]` syntax? This long list of mpirun arguments looks very funny :joy:

Reply:
It's supported, but I don't think there's any middle ground between one very long line in `[...]` format or the current approach that `yq` wouldn't mangle (leading to a hard-to-read diff). I agree it looks a bit odd though.